src/foam/u2/view/TableCellFormatter.js

```diff
@@ -194,8 +194,11 @@ foam.CLASS({
       class: 'foam.u2.view.TableCellFormatter',
       name: 'tableCellFormatter',
       value: function(date) {
-        // allow the browser to deal with this since we are technically using the user's preference
-        if ( date ) this.add(date.toLocaleDateString());
```

Comment:
When using foam.core.Date, we should always be displaying the UTC value. I'd suggest making this method do something like:
```
var d = new Date();
d.setFullYear(date.getUTCFullYear(), date.getUTCMonth(), date.getUTCDate());
this.add(d.toLocaleDateString());
```